jekyll/_cci2/build-config-cookbook.md

```diff
@@ -0,0 +1,334 @@
+
+### Prerequisites
+
+Before you can deploy any software changes to Amazon ECS on the CircleCI platform, you must first perform a series of configuration steps to ensure that you have properly set up your environment for the CircleCI platform. Also, because CircleCI has created an "orb" to simplify these steps, you will also need to ensure your instance has been configured for using CircleCI orbs.
+
+### Configuring Your Environment for the CircleCI Platform and Orbs
+
+1) Use CircleCI version 2.1 at the top of your `.circleci/config.yml` file.
+
+`version: 2.1`
+
+If you do not already have pipelines enabled, you'll need to go to **Project Settings -> Advanced Settings** and enable pipelines.
+
+2) Add the orbs stanza below your version, invoking the orb:
+
+```orbs:
+  aws-ecs: circleci/aws-ecs@0.0.6
+```
+
+3) Use `aws-ecs elements` in your existing workflows and jobs.
+
+## Updating the Amazon ECS Service
+
+Now that your environment is configured to work with orbs, you should update the Amazon ECS service to ensure you have the latest version of ECS. There are two different ways you can update the Amazon ECS Service, depending on whether you also want to update the existing Amazon Web Services CLI. Both of these approaches are described below.
+
+### Updating the Amazon ECS Sevice Without Updating AWS CLI
+
+If you want to update the Amazon ECS Service without updating the AWS CLI using CircleCI orbs, review the the yaml code sample shown below, which illustrates how you can update the ECS service.
+
+```yaml
+version: 2.1
+orbs:
+  aws-ecr: circleci/aws-ecr@0.0.4
+  aws-ecs: circleci/aws-ecs@0.0.3
+workflows:
+  build-and-deploy:
+    jobs:
+      - aws-ecr/build_and_push_image:
+          account-url: '${AWS_ACCOUNT_ID}.dkr.ecr.${AWS_REGION}.amazonaws.com'
+          repo: '${MY_APP_PREFIX}'
+          region: '${AWS_REGION}'
+          tag: '${CIRCLE_SHA1}'
+      - aws-ecs/deploy-service-update:
+          requires:
+            - aws-ecr/build_and_push_image
+          family: '${MY_APP_PREFIX}-service'
+          cluster-name: '${MY_APP_PREFIX}-cluster'
+          container-image-name-updates: 'container=${MY_APP_PREFIX}-service,tag=${CIRCLE_SHA1}'
+```
+
+Notice in this example that you need to instantiate two different AWS ECR orbs: `aws-ecr 0.0.3` and `aws-ecr 0.0.4` to update the ECS service. Once you have instantiated these two orbs, the orb workflows in the orb first build and push the image, and then deploy the service update to ECS.
```

Comment:
Should ecr = ecs here? and same for above ^^

Reply:
Oh no I guess it's the `aws-ecr 0.0.3` entry in the text that should be "ecs"

Reply:
change aws-ecr@0.0.3 to aws-ecs@0.0.3

Reply:
Fixed